ifrs17/Import/2ImportScope-PresentValue.ipynb

```diff
@@ -520,13 +520,13 @@
                 "public interface PresentValue : IWithGetValueFromValues {   ",
                 "\n    static ApplicabilityBuilder ScopeApplicabilityBuilder(ApplicabilityBuilder builder) =>",
                 "\n        builder.ForScope<PresentValue>(s => s",
-                "\n                                        .WithApplicability<ComputePresentValueWithIfrsVariable>(x => x.GetStorage().ImportFormat != ImportFormats.Cashflow || x.GetStorage().IsSecondaryScope(x.Identity.Id.DataNode))",
-                "\n                                        .WithApplicability<PresentValueFromDiscountedCashflow>(x => (x.Identity.Id.AocType == AocTypes.BOP && x.Identity.Id.Novelty != Novelties.C) || x.Identity.Id.AocType == AocTypes.EOP)",
-                "\n                                        .WithApplicability<CashflowAocStep>(x => x.Identity.Id.AocType == AocTypes.CF)",
-                "\n                                        .WithApplicability<PresentValueWithInterestAccretion>(x => x.Identity.Id.AocType == AocTypes.IA)",
-                "\n                                        .WithApplicability<EmptyValuesAocStep>(x => ComputationHelper.AocTypeWithNoPv.Contains(x.Identity.Id.AocType) ||",
-                "\n                                                                                    (x.Identity.Id.AocType == AocTypes.CRU && !x.GetStorage().GetCdr().Contains(x.Identity.AmountType)) )",
-                "\n                                       );",
+                "\n            .WithApplicability<ComputePresentValueWithIfrsVariable>(x => x.GetStorage().ImportFormat != ImportFormats.Cashflow || x.GetStorage().IsSecondaryScope(x.Identity.Id.DataNode))",
+                "\n            .WithApplicability<PresentValueFromDiscountedCashflow>(x => (x.Identity.Id.AocType == AocTypes.BOP && x.Identity.Id.Novelty != Novelties.C) || x.Identity.Id.AocType == AocTypes.EOP)",
+                "\n            .WithApplicability<CashflowAocStep>(x => x.Identity.Id.AocType == AocTypes.CF)",
+                "\n            .WithApplicability<PresentValueWithInterestAccretion>(x => x.Identity.Id.AocType == AocTypes.IA)",
+                "\n            .WithApplicability<EmptyValuesAocStep>(x => ComputationHelper.AocStepWithNoPv.Contains(new AocStep(x.Identity.Id.AocType, x.Identity.Id.Novelty)) ||",
```

Comment:
Just checking, AocStepWithNoPv is a Hashset?

Reply:
yes, it is.